### DISCUSSION TITLE
How to add cards in docusaurus site just like implemented in playground page in docusaurus site?

### DISCUSSION THREAD

**Discussion by RudraSen2** [facebook/docusaurus#7708] [Q&A] [answered]
I want to add cards to my docusaurus v2 site just like in the Playground Page in docusaurus.io/docs/playground. Can someone tell me how to implement it?

**Comment by slorber** [comment, marked as answer]
This is a custom React component added with MDX, not a Docusaurus feature.

You would have to copy our React code to use this on your site:
- https://github.com/facebook/docusaurus/blob/main/website/docs/playground.mdx
- https://github.com/facebook/docusaurus/blob/main/website/src/components/Playground/index.tsx

**Reply by RudraSen2** [reply]
I tried this. It gives me unknown errors.

You can see it live here: https://stackblitz.com/edit/github-gf4n6a?file=src/components/Playground/index.tsx

**Reply by slorber** [reply]
Implementing complex features require knowledge on frontend development and React.

Looking at your code, it looks like you lack basic knowledge on bundlers, React, JS and TS. Unfortunately, it is outside the scope of Docusaurus to teach you these concepts.

I can only give you some basic hints, but ultimately you'll have to learn the real frontend skills to become productive:
- you should not use TS types and interfaces in JS files
- you can't require image URLs, it's not supported by frontend bundlers

**Reply by RudraSen2** [reply]
I'm getting error when adding `mdx-code-block` in `playground.mdx`. Can you help me with that?

https://stackblitz.com/edit/github-gf4n6a?file=docs/playground.mdx

**Reply by RudraSen2** [reply]
Now it's working! Thanks for any help!

https://stackblitz.com/edit/github-gf4n6a
